RecoTracker/FinalTrackSelectors/plugins/powN.h

```diff
@@ -0,0 +1,39 @@
+namespace {
```

Comment:
better also include this file in RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc as well.
So that we don't proliferate copies of the same implementation.